_posts/2016-04-07-developing-nectar-extensions-part-1.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+cover: 'assets/images/general-cover-3.jpg'
+title: Developing Nectar Extensions Part 1
+tags: docs
+subclass: 'post tag-docs'
+categories: 'elixir'
+author: 'Nimish'
+navigation: true
+logo: 'assets/images/nectar-cart.png'
+---
+
+>
+The post belongs to _NectarCommerce and Extension Framework Awareness_ Series
+>
+1. _[NectarCommerce Vision](http://vinsol.com/blog/2016/04/08/nectarcommerce-vision/)_
+1. _[Extension Framework Game Plan](http://vinsol.com/blog/2016/04/12/extension-framework-game-plan/)_
+1. _[Introduction to Metaprogramming](http://vinsol.com/blog/2016/04/14/introduction-to-metaprogramming/)_
+1. _[Ecto Model Schema Extension](http://vinsol.com/blog/2016/04/15/ecto-model-schema-extension/)_
+1. Ecto Model Support Functions Extension
+1. Phoenix Router Extension
+1. Phoenix View Extension
+1. Running Multiple Elixir Apps Together
+1. Extension Approach Explained
+1. **Developer Experience and Workflow developing Favorite Product Extension**
+1. Developer Experience and Workflow testing Favorite Product Extension
+
+
+
+Developing NectarCommerce Extensions Part 1
+=================
+
+
+### Where we left off ###
+
+
+In the past few blogs we have learned how to write code that extends existing models, routers, added methods to override view rendering and run multiple phoenix application together in the same umbrella project. Let's Continue to build upon that and write our first extension for NectarCommerce [favorite products](http://vinsol.com/blog/2016/04/12/extension-framework-game-plan/) and ultimately our store based on NectarCommerce.
```

Comment:
multiple phoenix **applications** ..


ultimately **for** our store based on NectarCommerce.